Views/ItemVM/Index.cshtml

```diff
@@ -0,0 +1,43 @@
+﻿@model IEnumerable<HowManyCaloriesIn.Models.ItemViewModel>
```

Comment:
This is where the error is happening. The view is expecting a list of `ItemViewModels` but nothing is being returned so the model is `null`.